vmf/scripts/mods/vmf/modules/core/network.lua

```diff
@@ -289,7 +293,48 @@ local function vmf_network_recv(sender, channel_id, rpc_data1, rpc_data2)
         end
       end
     end)
+end
 
+local expected_pong_data_blocks = {}
+local partial_pong_mod_data = {}
+local partial_pong_rpc_data = {}
```

Comment:
The convention for file upvalues in VMF is to have them at the top of the file and prefix them with `_` like `_partial_pong_rpc_data`, etc.

Reply:
Done